lms/static/sass/xblocks/problem_builder.scss

```diff
@@ -1,25 +1,5 @@
 // Overrides for Problem Builder
 // from https://github.com/open-craft/problem-builder/blob/7e76920/problem_builder/public/themes/apros.css
-.mentoring {
-    .questionnaire {
-        .choice-tips, .feedback {
-            // Avoid a global reset to border-box found on Apros
-            box-sizing: content-box;  
-        }
-        .choice-label {
```

Comment:
You did not move over this code, any reason why?

Reply:
This is the main fix for the issue showcased on the screenshots on the PR description